tests/lib/managers/terms-of-service-test.js

```diff
@@ -0,0 +1,571 @@
+				id: '1234',
+				status: 'enabled',
+				enterprise: {},
+				tos_type: 'external',
+				text: 'TEST',
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.stub(boxClientFake, 'post').returns(Promise.resolve(response));
+			return termsOfService.create('managed', 'enabled', 'Test Text')
+				.then(termsOfServiceObject => {
+					assert.strictEqual(termsOfServiceObject, response, 'terms of service object is returned');
+				});
+		});
+	});
+
+	describe('update()', function() {
+		it('should make PUT request to update custom terms of service info when called', function() {
+			var expectedTermsOfServiceParam = { body: { status: 'enabled', text: 'Test Text'}};
+
+			var response = {
+				type: 'terms_of_service',
+				id: '1234',
+				status: 'enabled',
+				enterprise: {},
+				tos_type: 'managed',
+				text: 'Test Text',
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.mock(boxClientFake).expects('put').withArgs('/terms_of_services/1234', expectedTermsOfServiceParam)
+				.returns(Promise.resolve(response));
+			termsOfService.update(TERMS_OF_SERVICE_ID, 'enabled', 'Test Text');
+		});
+
+		it('should call callback with the updated terms of service info when a 200 response is returned', function(done) {
+			var response = {
+				type: 'terms_of_service',
+				id: '1234',
+				status: 'enabled',
+				enterprise: {},
+				tos_type: 'external',
+				text: 'Test Text',
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.stub(boxClientFake, 'put').yieldsAsync(null, response);
+			termsOfService.update(TERMS_OF_SERVICE_ID, 'enabled', 'Test Text', function(err, termsOfServiceObject) {
+				assert.ifError(err);
+				assert.strictEqual(termsOfServiceObject, response, 'terms of service object is returned');
+				done();
+			});
+		});
+
+		it('should return a promise resolving to updated terms of service object when a 200 response is returned', function() {
+			var response = {
+				type: 'terms_of_service',
+				id: '1234',
+				status: 'enabled',
+				enterprise: {},
+				tos_type: 'external',
+				text: 'Test Text',
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.stub(boxClientFake, 'put').returns(Promise.resolve(response));
+			return termsOfService.update(TERMS_OF_SERVICE_ID, 'enabled', 'Test Text')
+				.then(termsOfServiceObject => {
+					assert.strictEqual(termsOfServiceObject, response, 'terms of service object is returned');
+				});
+		});
+	});
+
+	describe('createUserStatuses()', function() {
```

Comment:
When you change the name of this method to make it singular, remember to change it here too!